miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppSdkConfig.kt

```diff
@@ -3,34 +3,44 @@ package com.rakuten.tech.mobile.miniapp
 /**
  * This represents the configuration settings for the Mini App SDK.
  * @property baseUrl Base URL used for retrieving a Mini App.
- * @property rasAppId App ID for the Platform API.
+ * @property rasProjectId Project ID for the Platform API.
+ * @property rasAppId App ID for the Platform API (Deprecated).
  * @property subscriptionKey Subscription Key for the Platform API.
  * @property hostAppVersionId Version of the host app, used to determine feature compatibility for Mini App.
  * @property hostAppUserAgentInfo User Agent information from Host App.
  * @property isTestMode Whether the sdk is making use of the Test API Endpoints for under "Testing" mini apps.
  */
 data class MiniAppSdkConfig(
     val baseUrl: String,
-    val rasAppId: String,
+    var rasProjectId: String,
+    @Deprecated("Use rasProjectId") var rasAppId: String,
```

Comment:
This one is a public class, so I think there could be a couple of problems:
- The host app will always have to define both `rasAppId` and `rasProjectId`.
- This is technically a breaking change if the host app didn't use the parameterized names (though that's unlikely since there are so many parameters 😆. But this would be the case in a Java project).

So how about something like this?
```
data class MiniAppSdkConfig(
    val baseUrl: String,
    val rasProjectId: String,
    val subscriptionKey: String,
    val hostAppVersionId: String,
    val hostAppUserAgentInfo: String,
    val isTestMode: Boolean
) {    
    @Deprecated("Use rasProjectId instead of rasAppId")
    constructor(
        baseUrl: String,
        rasAppId: String,
        rasProjectId: String = rasAppId,
        subscriptionKey: String,
        hostAppVersionId: String,
        hostAppUserAgentInfo: String,
        isTestMode: Boolean
    ) : this(
        baseUrl = baseUrl,
        rasProjectId = rasProjectId,
        subscriptionKey = subscriptionKey,
        hostAppVersionId = hostAppVersionId,
        hostAppUserAgentInfo = hostAppUserAgentInfo,
        isTestMode = isTestMode
    )
// ...
}
```

Reply:
Thank you, I think this is the correct approach. I have updated this part.